app/grandchallenge/pages/templates/pages/challenge_settings_base.html

```diff
@@ -29,6 +29,11 @@
                     href="{% url 'challenge-update' challenge_short_name=challenge.short_name %}">
                       <i class="fas fa-cog fa-fw"></i> General Settings</a>
               </li>
+              <li class="nav-item">
+                <a class="nav-link px-4 py-1 mb-1 {% if request.resolver_match.view_name == 'participants:registration-question-list' or request.resolver_match.view_name == 'participants:registration-question-create' or request.resolver_match.view_name == 'participants:registration-question-delete'  %}active{% endif %}"
+                  href="{% url 'participants:registration-question-list' challenge_short_name=challenge.short_name %}">
+                    <i class="fas fa-question-circle fa-fw"></i> Registration Questions </a>
+              </li>
```

Comment:
Maybe use `{% comment %}` for now to hide it whilst the feature is incomplete?

Reply:
Good idea! Sort of an ad hoc, but very easy to remove feature flag.